src/encode_task_filter.py

```diff
@@ -255,13 +229,12 @@ def pbc_qc_se(bam, mito_chr_name, out_dir):
 def pbc_qc_pe(bam, mito_chr_name, nth, out_dir):
     prefix = os.path.join(out_dir,
         os.path.basename(strip_ext_bam(bam)))
-    pbc_qc = '{}.pbc.qc'.format(prefix)
+    pbc_qc = '{}.lib_complexity.qc'.format(prefix)
 
-    # nmsrt_bam = samtools_name_sort(bam, nth, out_dir)
-    nmsrt_bam = sambamba_name_sort(bam, nth, out_dir)
+    nmsrt_bam = samtools_name_sort(bam, nth, out_dir)
     cmd3 = 'bedtools bamtobed -bedpe -i {} | '
     cmd3 += 'awk \'BEGIN{{OFS="\\t"}}{{print $1,$2,$4,$6,$9,$10}}\' | '
-    cmd3 += 'grep -v "^{}\\b" | sort | uniq -c | '
+    cmd3 += 'grep -v "^{}\\s" | sort | uniq -c | '
```

Comment:
I thought `\b` is too ambiguous for our purpose (finding a end boundary (\t and whitespace only) for a chromosome name).

Yes you are right. Similar BASH command lines appear in the whole pipeline because this pipeline is based on a [spec document](https://docs.google.com/document/d/1lG_Rd7fnYgRpSIqrIfuVlAz2dW1VaSQThzk836Db99c/edit#) and pipeline's python script for each task is simply a re-write of these command lines in the spec doc. I will keep it as is for now.